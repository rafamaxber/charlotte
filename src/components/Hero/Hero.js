import React from 'react';
import styled from 'styled-components';
import banner from '../../assets/hero.jpg';

const Hero = styled.div`
  position: absolute;
  width: 100%;
  height: 300px;
  left: 0px;
  top: 0px;
  background-position: top center;
  background-repeat: no-repeat;
  background-image: url(${banner});
  background-size: cover;
  z-index: -1;
  
  @media (min-width: 570px) {
    height: 600px;
    background-size: 100%;
    background: palevioletred;
  } 
`;

export default () => (
  <Hero>
    
  </Hero>
);