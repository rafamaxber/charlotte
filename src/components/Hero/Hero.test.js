import React from 'react';
import renderer from 'react-test-renderer';
import Hero from './Hero';

describe('Hero', () => {
  it('Should match with snapshot', () => {
    const component = renderer.create(
      <Hero></Hero>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
