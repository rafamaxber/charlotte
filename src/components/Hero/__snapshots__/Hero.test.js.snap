// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero Should match with snapshot 1`] = `
<div
  className="sc-bdVaJa jsknkD"
>
  <div
    className="sc-bwzfXH kqucqH"
  >
    <img
      className="sc-htpNat hjVsta"
      src="crown.svg"
    />
    <div
      className="sc-bxivhb eKbCDM"
    >
      Welcome to
    </div>
    <div
      className="sc-ifAKCX geNkme"
    >
      Charlotte
    </div>
    <div
      className="sc-bxivhb eKbCDM"
    >
      the queen city
    </div>
  </div>
</div>
`;
